### DISCUSSION TITLE
Get items for the `Open World Survival Crafting Fest` 2024

### DISCUSSION THREAD

**Discussion by xTCry** [chr233/ASFEnhance#261] [杂谈 / General]
**Free - Valid only to June 3rd**

### Use command

`RPI [Bots] 288428,288430,288429`

### Items

 - `288428` - **Richious gem** <img src="https://cdn.cloudflare.steamstatic.com/steamcommunity/public/images/items/2855130/bb0f61fedbaa52bfcea86ca1a6ca35d114c01a50.png" height="90" />
 - `288430` - **Nailed it** <img src="https://cdn.cloudflare.steamstatic.com/steamcommunity/public/images/items/2855130/7cd2db45bb4b7fb7554c1424fc9316d3ebf817ed.png" height="90" />
 - `288429` - **Survivin' and thrivin'** <img src="https://cdn.cloudflare.steamstatic.com/steamcommunity/public/images/items/2855130/2214939504447ffc9c4d3257004a84da037863a9.gif" height="90" />

[View more on SteamDB](https://steamdb.info/app/2855130/history/)

**Comment by woctezuma** [comment]
Thank you! 🥳
